.drone.script

```diff
@@ -1,6 +1,7 @@
 def main():
   return [
       # tf_version, ubuntu_version, cuda_version, bazel_version, go_version
+      build_pipeline_new("1.14.0", "16.04", "10.0", "0.24.1", "1.11"),
```

Comment:
Can we treat tf 1.14 in a different PR? There are more changes necessary. Would be nice if this PR is just about simplifying the CMake Lists.

Could you please rebase on on the master branch instead of picking up not working changes from tf1.14.